core/iwasm/libraries/lib-socket/inc/wasi_socket_ext.h

```diff
@@ -336,7 +363,7 @@ __imported_wasi_snapshot_preview1_sock_get_reuse_port(int32_t arg0,
                    __import_name__("sock_get_reuse_port")));
 
 static inline __wasi_errno_t
-__wasi_sock_get_reuse_port(__wasi_fd_t fd, int8_t *reuse)
+__wasi_sock_get_reuse_port(__wasi_fd_t fd, uint8_t *reuse)
```

Comment:
is there a particular reason about from `int8_t` to `uint8_t`

Reply:
Only reason is that for other functions I had used `uint8_t` for the enable/disable flag and I thought I would change this for consistency with those other set/get sock opts